### ISSUE TITLE
expo-cli fails install on Window 10

### ISSUE THREAD

**Issue by Bilal-S** [expo/expo-cli#1297]
## Description

expo-cli 3.9.1 fails to install on Windows 10.

npm: 6.9.0
node: 10.16.0

Ran: 
npm install -g expo-cli

Error returned 
```
npm ERR! errno 1
npm ERR! envsub@3.1.0 postinstall: `test -d .git && cp gitHookPrePush.sh .git/hooks/pre-push || true`
npm ERR! Exit status 1
```

## Expected Behavior

Install completes successfully

## action taken

We reverted to version 3.5.0 which installed without any errors.

**Comment by mpcen** [comment]
Duplicate of issue [1284](https://github.com/expo/expo-cli/issues/1284). There is a workaround posted on there.

**Comment by Arlen22** [comment]
Also, since this is windows 10, you could install the Ubuntu VM inside windows that Microsoft has now included in Windows 10, but that wouldn’t accomplish the same purpose and probably would not work cross-platform even if you use Linux to install it in the windows NPM directory because it would do a a Linux platform install.

**Comment by cbdeveloper** [comment]
I have this same issue.

![image](https://user-images.githubusercontent.com/43407798/70128433-b19b8c00-1674-11ea-82e3-fe9362114eb9.png)

**Comment by cbdeveloper** [comment]
SOLVED! Followed the workaround from https://github.com/expo/expo-cli/issues/1284 and was able to install using Git Bash for Windows (MINGW64) without any errors.

Before I was using Powershell from VSCode.

**Comment by kaisanthia01** [comment]
How to fix
![Error](https://user-images.githubusercontent.com/53453870/70210065-5943b780-1764-11ea-8c66-482b8adb32ab.png)

**Comment by brentvatne** [comment]
@kaisanthia01 - that is not a blocking installation error, it tries to install sharp and if it doesn't work then it will proceed anyways

**Comment by m-elgamal** [comment]
Doesn't solve it for me, tried MinGw64 git bash with npm install -g expo-cli  and gives the same error: 
`npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ejs@2.7.4 postinstall: `node ./postinstall.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ejs@2.7.4 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\TGN\AppData\Roaming\npm-cache\_logs\2019-12-06T02_07_31_667Z-debug.log`

**Comment by nebmesemaou** [comment]
can not use expo cli anymore same problem here.

**Comment by brentvatne** [comment]
this should be resolved on 3.10.2. the issue was that @expo/build-tools was installing a library called envsub that would fail on windows 10 in powershell/cmd.exe, so we switched that to an optional dependency - you will still see that envsub failed to install but the installation of expo-cli should succeed. we're going to consider breaking @expo/build-tools and similar functionality into a separate cli tool to avoid issues like this in the future.

**Comment by m-elgamal** [comment]
Just tried it, Fixed
